tools/visual-pipeline-and-platform-evaluation-tool/home_page.py

```diff
@@ -0,0 +1,82 @@
+import gradio as gr
+
+from gstpipeline import PipelineLoader
+from device import DeviceDiscovery
+
+with gr.Blocks() as home:
+    # Header
+    gr.HTML(
+        "<div class='spark-header'>"
+        "  <div class='spark-header-line'></div>"
+        "  <img src='https://www.intel.com/content/dam/logos/intel-header-logo.svg' class='spark-logo'></img>"
+        "  <div class='spark-title'>Visual Pipeline and Platform Evaluation Tool</div>"
+        "</div>"
+    )
+
+    gr.Markdown(
+        """
+        ## Recommended Pipelines
+
+        Below is a list of recommended pipelines you can use to evaluate video analytics performance.
+        Click on "Configure and Run" to get started with customizing and benchmarking a pipeline for your
+        use case.
+        """
+    )
+
+    with gr.Row():
+        for pipeline in PipelineLoader.list():
+            pipeline_info = PipelineLoader.config(pipeline)
+
+            with gr.Column(scale=1, min_width=100):
+                gr.Image(
+                    value=lambda x=pipeline: f"./pipelines/{x}/thumbnail.png",
+                    show_label=False,
+                    show_download_button=False,
+                    show_fullscreen_button=False,
+                    interactive=False,
+                    width=710,
+                )
+
+                gr.Markdown(
+                    f"### {pipeline_info['name']}\n{pipeline_info['definition']}"
+                )
+
+                is_enabled = pipeline_info.get("metadata", {}).get("enabled", False)
+
+                gr.Button(
+                    value=("Configure and Run" if is_enabled else "Coming Soon"),
+                    elem_classes="configure-and-run-button",
+                    interactive=is_enabled,
+                    link=f"/{pipeline}",
```

Comment:
It looks like those links don’t pass the theme settings:
If I go to the address: `http://localhost:7860/?__theme=light` and click on any of the pipelines (either in the tab or in “Configure and Run”), it redirects me to the page in format `http://localhost:7860/simplevs`.
That page’s address doesn’t contain the theme settings, so in my case (when the system and browser default to dark theme), the pipeline page will be dark.